build/generate_bundle.sh

```diff
@@ -34,6 +34,18 @@ generate_bundle() {
     popd > /dev/null 2>&1
 
     sed -i -E "${REPLACE_REGEX}" "${WORKING_DIR}/manifests/${OPERATOR_NAME}.clusterserviceversion.yaml"
+    cat >> "${WORKING_DIR}/metadata/annotations.yaml" << EOF
+  features.operators.openshift.io/disconnected: "false"
+  features.operators.openshift.io/fips-compliant: "false"
+  features.operators.openshift.io/proxy-aware: "false"
+  features.operators.openshift.io/cnf: "false"
+  features.operators.openshift.io/cni: "false"
+  features.operators.openshift.io/csi: "false"
+  features.operators.openshift.io/tls-profiles: "false"
```

Comment:
That's an RFE I've never considered before, but I can certainly see why it would be desirable!